cmd/chaos-controller-manager/main.go

```diff
@@ -97,7 +98,8 @@ type RunParams struct {
 	DaemonClientBuilder *chaosdaemon.ChaosDaemonClientBuilder
 	MetricsCollector    *metrics.ChaosControllerManagerMetricsCollector
 
-	Objs []types.Object `group:"objs"`
+	Objs        []types.Object        `group:"objs"`
+	WebhookObjs []types.WebhookObject `group:"webhookObjs"`
```

Comment:
`types.WebhookObject` has a field `v1alpha1.WebhookObject`, which is the interface I created for some crd that only implement `webhook.Validator` and `webhook.Defaulter` interface, but not implement `v1alpha1.InnerObject` interface. 

For example, the new crd `PhysicalMachine` is not an experiment, it cannot implement `v1alpha1.InnerObject` interface, so here it would be the `webhookObjs`.